cmd/bosun/conf/system_elastic_all.go

```diff
@@ -0,0 +1,49 @@
+package conf
+
+import (
+	"fmt"
+
+	"bosun.org/cmd/bosun/expr"
+)
+
+// ParseESConfig return expr.ElasticHost
+func parseESConfig(sc *SystemConf) expr.ElasticHosts {
+	store := make(map[string]expr.ElasticConfig)
+	for hostPrefix, value := range sc.ElasticConf {
+		// build es config per cluster
+		var cfg expr.ElasticConfig
+		switch expr.ESVersion(value.Version) {
+		case expr.ESV2:
+			cfg = parseESConfig2(value)
+		case expr.ESV5:
+			cfg = parseESConfig5(value)
+		case expr.ESV6:
+			cfg = parseESConfig6(value)
+		default:
+			panic(fmt.Errorf("conf: invalid elastic version: %s (supported versions are: v2, v5, v6)", value.Version))
```

Comment:
slog.Fatal is more appropriate (exit with error code, no stack trace). panic is more for programming errors and not user configuration errors.